saleor/settings.py

```diff
@@ -598,6 +598,9 @@ def get_host():
 # Slug for channel precreated in Django migrations
 DEFAULT_CHANNEL_SLUG = os.environ.get("DEFAULT_CHANNEL_SLUG", "default-channel")
 
+# Set this to `True` if you want to skip creating default objects in database.
+BLOCK_POPULATE_DEFAULTS = False
```

Comment:
Wouldn't be good to use os.environ here also?

Reply:
Maybe a name like `CREATE_BASE_STORE_DATA` would be more meaningful?
WDYT @maarcingebala @jakubkuc

Reply:
Yes, I would also revert the condition, it's more intuitive to me if things are enabled by default, I would change it to `POPULATE_DEFAULTS = True`. We can use os.environ as well.